### DISCUSSION TITLE
repeat texture op - add checkbox for mirror repeat

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#361] [Feature Requests]
be great if rather than repeating we could also unfold an image

rather than this
<img width="492" alt="Screenshot 2022-09-29 at 14 35 10" src="https://user-images.githubusercontent.com/36418408/193046173-c06d4b05-e6d5-4813-99c2-4e458407c410.png">
it would be this
<img width="491" alt="Screenshot 2022-09-29 at 14 35 20" src="https://user-images.githubusercontent.com/36418408/193046212-58529f91-4cc6-4518-b28c-aa7e18488dbb.png">

**Comment by pandrr** [comment]
post example patch

probably you have to set it in the imagecompose
